lib/src/wavy.dart

```diff
@@ -232,8 +232,8 @@ class _WTextPainter extends CustomPainter {
 
   void calculateMove() {
     double height = _textLayoutInfo[0].height;
-    double _percent = progress - progress.floor();
     int txtInMoInd = progress.floor();
+    double _percent = progress - txtInMoInd;
```

Comment:
Sorry I didn't notice this before, but a local variable should not have an underscore prefix -- that's confusing.

Reply:
Will do this too